Logging/AdaptiveHandler.php

```diff
@@ -28,13 +28,4 @@ public function __construct(
     ) {
         parent::__construct($level, $bubble, $appName, $explodeArrays, $transactionName);
     }
-
-    protected function write(LogRecord $record): void
```

Comment:
The parent has a different signature in different versions, but the implementation is the same anyway. So just use the parent method.